# Array of GloveIMU messages with a header for global reference

std_msgs/Header header

# Acquisition time
duration acquisition_time

# Accelerometers and gyroscopes data
GloveIMU[] data
